### ISSUE TITLE
Error building docker image

### ISSUE THREAD

**Issue by fzwain** [jdekarske/HOMESTRI-dragdrop#1]
I cloned the repo and tried to build the docker image but it failed building the image with the following error on the last command of the docker file. 

This is the command I used to try to build:
docker build -t gui .

Here is the issue that I get:
 ```
=> ERROR [5/5] RUN npm i && npx webpack                                                                          13.8s
------
 > [5/5] RUN npm i && npx webpack:
#7 12.36
#7 12.36 added 566 packages, and audited 567 packages in 12s
#7 12.36
#7 12.36 108 packages are looking for funding
#7 12.36   run `npm fund` for details
#7 12.36
#7 12.36 3 vulnerabilities (1 high, 2 critical)
#7 12.36
#7 12.36 To address all issues, run:
#7 12.36   npm audit fix
#7 12.36
#7 12.36 Run `npm audit` for details.
#7 12.36 npm notice
#7 12.36 npm notice New major version of npm available! 8.1.0 -> 9.1.2
#7 12.36 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.1.2>
#7 12.36 npm notice Run `npm install -g npm@9.1.2` to update!
#7 12.36 npm notice
#7 13.16 assets by status 0 bytes [cached] 1 asset
#7 13.16
#7 13.16 WARNING in configuration
#7 13.16 The 'mode' option has not been set, webpack will fallback to 'production' for this value.
#7 13.16 Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
#7 13.16 You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
#7 13.16
#7 13.16 ERROR in main
#7 13.16 Module not found: Error: Can't resolve './src' in '/home'
#7 13.16 resolve './src' in '/home'
#7 13.16   using description file: /home/package.json (relative path: .)
#7 13.16     Field 'browser' doesn't contain a valid alias configuration
#7 13.16     using description file: /home/package.json (relative path: ./src)
#7 13.16       no extension
#7 13.16         Field 'browser' doesn't contain a valid alias configuration
#7 13.16         /home/src is not a file
#7 13.16       .js
#7 13.16         Field 'browser' doesn't contain a valid alias configuration
#7 13.16         /home/src.js doesn't exist
#7 13.16       .json
#7 13.16         Field 'browser' doesn't contain a valid alias configuration
#7 13.16         /home/src.json doesn't exist
#7 13.16       .wasm
#7 13.16         Field 'browser' doesn't contain a valid alias configuration
#7 13.16         /home/src.wasm doesn't exist
#7 13.16       as directory
#7 13.16         existing directory /home/src
#7 13.16           using description file: /home/package.json (relative path: ./src)
#7 13.16             using path: /home/src/index
#7 13.16               using description file: /home/package.json (relative path: ./src/index)
#7 13.16                 no extension
#7 13.16                   Field 'browser' doesn't contain a valid alias configuration
#7 13.16                   /home/src/index doesn't exist
#7 13.16                 .js
#7 13.16                   Field 'browser' doesn't contain a valid alias configuration
#7 13.16                   /home/src/index.js doesn't exist
#7 13.16                 .json
#7 13.16                   Field 'browser' doesn't contain a valid alias configuration
#7 13.16                   /home/src/index.json doesn't exist
#7 13.16                 .wasm
#7 13.16                   Field 'browser' doesn't contain a valid alias configuration
#7 13.16                   /home/src/index.wasm doesn't exist
#7 13.16
#7 13.16 webpack 5.72.1 compiled with 1 error and 1 warning in 136 ms
------
executor failed running [/bin/sh -c npm i && npx webpack]: exit code: 1
```

**Comment by jdekarske** [comment]
fixed in 1479e3f

**Comment by fzwain** [comment]
Hi Jason, Thanks for getting back to me, I am now able to get the interface up and running. Best,Faress Sent from Mail for Windows From: Jason DekarskeSent: Saturday, November 19, 2022 1:56 PMTo: jdekarske/HOMESTRI-dragdropCc: fzwain; AuthorSubject: Re: [jdekarske/HOMESTRI-dragdrop] Error building docker image (Issue #1) fixed in 1479e3f—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
